tools/ow-utils/Dockerfile.arm

```diff
@@ -0,0 +1,80 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM arm64v8/eclipse-temurin:8u422-b05-jdk-noble
```

Comment:
I only changed the base image from `adoptopenjdk/openjdk8:jdk8u262-b10` to `arm64v8/eclipse-temurin:8u422-b05-jdk-noble`.

I used it as the current image is deprecated.

<img width="784" alt="image" src="https://github.com/user-attachments/assets/2f629baa-7438-4f43-ae55-f984b7027483">

https://hub.docker.com/_/adoptopenjdk